http/jetty12/src/main/java/org/apache/felix/http/jetty/internal/JettyService.java

```diff
@@ -564,6 +566,18 @@ private void configureHttpConnectionFactory(HttpConnectionFactory connFactory)
         config.setResponseHeaderSize(this.config.getHeaderSize());
         config.setOutputBufferSize(this.config.getResponseBufferSize());
 
+        String uriComplianceMode = this.config.getProperty(JettyConfig.FELIX_JETTY_URI_COMPLIANCE_MODE, null);
+        if (uriComplianceMode != null) {
+            config.setUriCompliance(UriCompliance.valueOf(uriComplianceMode));
```

Comment:
If the value is not one from the enum, this will throw an exception, I think we should catch this, log and use the default

Reply:
Good one, will adjust.